src/send/SendConfirmation.tsx

```diff
@@ -373,6 +388,13 @@ const styles = StyleSheet.create({
     color: colors.infoDark,
     paddingRight: 8,
   },
+  subHeading: {
+    marginVertical: 0,
+  },
+  subHeadingText: {
+    ...fontStyles.small,
```

Comment:
Used deprecated fontStyles as mock aren't updated and to match subHeading of line below.

Reply:
Do we want to add a clean up note here or check with Nitya about changing the mocks? I worry that if we add the deprecated style here in the future when we change this screen we'll be in the same boat.

Reply:
@finnian0826 unfortunately It's already used in several other places throughout `src/send/SendConfirmation.tsx`; however, I do think it's worthwhile to call out uses of the old fonts on new screens and new mocks.

Reply:
is there no equivalent style in typescale? Or is the concern that there's multiple other uses and hard to update now?

Reply:
My main concern is that there are several usages of `fontStyle` within the file. I'll switch them over to typeScale and see if I can get a similar appearance.

Reply:
Was able to update the styles in 55bcde12049b9f9b5fd74fe7e4c6bcdc5f3b2ee1! The snapshots were pretty helpful in the case to make sure the new fonts were as close to the old ones as possible.